.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,49 @@
+name: Bug Report
+description: File a bug report
+title: ":bug: "
+labels: ["Type · Bug", "Status · Unconfirmed"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        We appreciate you taking the time to fill out this bug report!
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: What issue are you facing?
+      placeholder: Tell us what you see!
+    validations:
+      required: true
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: What should have happened?
+      description: What were you trying to accomplish?
+      placeholder: Tell us what you expected to see!
+    validations:
+      required: true
+  - type: dropdown
+    id: browsers
+    attributes:
+      label: What browsers are you seeing the problem on?
+      multiple: true
+      options:
```

Comment:
Could we add device as another lever and/or integrate it into browser?

Reply:
We certainly can. I'll add that as a new field!